src/WpfMath/Parsers/StandardCommands.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections.Generic;
+using WpfMath.Atoms;
+
+namespace WpfMath.Parsers
+{
+    internal static class StandardCommands
+    {
+        private class UnderlineCommand : ICommandParser
+        {
+            public CommandProcessingResult ProcessCommand(CommandContext context)
+            {
+                var source = context.FormulaSource;
+                var position = context.ArgumentsStartPosition;
+                var underlineFormula = context.Parser.Parse(
+                    TexFormulaParser.ReadElement(source, ref position),
+                    context.Formula.TextStyle);
+                var start = context.CommandNameStartPosition;
+                var atomSource = source.Segment(start, position - start);
+                var atom = new UnderlinedAtom(atomSource, underlineFormula.RootAtom);
+                return new CommandProcessingResult(atom, position);
+            }
```

Comment:
In the future, I want to make a base class for these simple parsers and move the common parts into it. But this will work for now.